assets/pages/bundles-landing/support-landing-ab-test/supportLanding.scss

```diff
@@ -208,11 +208,305 @@
     // ----- Contribute
 
     .contribute-new-design {
-      background-color: darken(gu-colour(multimedia-main-2), 5%);
+      background-color: darken(#fff, 5%);
     }
 
     .contribute-new-design__content {
-      background-color: gu-colour(multimedia-main-2);
+      background-color: #fff;
+      border-color: gu-colour(light-yellow);
+      color: gu-colour(neutral-2);
+      position: relative;
+
+      .component-info-section__header {
+        position: relative;
+
+        .component-secure {
+          position: absolute;
+          top: 10px;
```

Comment:
Is it the same that `$gu-v-spacing / 2` ?

Reply:
Nope, it's the same as `$gu-h-spacing / 2`, but this isn't a horizontal spacing.